.github/dependabot.yml

```diff
@@ -0,0 +1,9 @@
+version: 2
+updates:
+  - package-ecosystem: "pub"
+    directory: "/"
```

Comment:
Question: Is this required?

Reply:
yes it is required https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file

<img width="807" alt="Screenshot 2024-12-13 at 12 15 05 PM" src="https://github.com/user-attachments/assets/a0febbbe-2748-4e84-919e-9f4679abdc4e" />